README.md

```diff
@@ -16,3 +16,79 @@ Create a `.env` file. See `.env.example` for environment variables which may be
+| CPKFactory | [0x336c19296d3989e9e0c2561ef21c964068657c38](https://rinkeby.etherscan.io/address/0x336c19296d3989e9e0c2561ef21c964068657c38#code) |
+| Uniswap V2 Factory | [0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f](https://rinkeby.etherscan.io/address/0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f#code) |
+| WETH9 | [0xc778417E063141139Fce010982780140Aa0cD5Ab](https://rinkeby.etherscan.io/address/0xc778417E063141139Fce010982780140Aa0cD5Ab#code) |
+| GelatoCore | [0x733aDEf4f8346FD96107d8d6605eA9ab5645d632](https://rinkeby.etherscan.io/address/0x733aDEf4f8346FD96107d8d6605eA9ab5645d632#code) |
+
+
+## xDai Network
+
+| Contract | Address |
+| -------- | -------- |
+| [ConditionalTokens](https://github.com/gnosis/conditional-tokens-contracts) | [0xCeAfDD6bc0bEF976fdCd1112955828E00543c0Ce](https://blockscout.com/poa/xdai/address/0xCeAfDD6bc0bEF976fdCd1112955828E00543c0Ce/read-contract)     |
+| [FPMMDeterministicFactory](https://github.com/gnosis/conditional-tokens-market-makers) | [0x9083A2B699c0a4AD06F63580BDE2635d26a3eeF0](https://blockscout.com/poa/xdai/address/0x9083A2B699c0a4AD06F63580BDE2635d26a3eeF0/contracts) |
+| [Kleros bridge](https://github.com/kleros/cross-chain-realitio-proxy) | [0xe40DD83a262da3f56976038F1554Fe541Fa75ecd](https://blockscout.com/poa/xdai/address/0xe40DD83a262da3f56976038F1554Fe541Fa75ecd/contracts) |
+| [Reality.eth](https://github.com/realitio/realitio-contracts) | [0x79e32aE03fb27B07C89c0c568F80287C01ca2E57](https://blockscout.com/poa/xdai/address/0x79e32aE03fb27B07C89c0c568F80287C01ca2E57/contracts) |
+| Reality.eth Oracle Adapters | [0x2bf1BFb0eB6276a4F4B60044068Cb8CdEB89f79B](https://blockscout.com/poa/xdai/address/0x2bf1BFb0eB6276a4F4B60044068Cb8CdEB89f79B/contracts) |
+| Reality.eth Oracle Adapters (Scalar) | [0xb97FCb6adf4c4aF9981932a004e6CC47173d0Bfc](https://blockscout.com/poa/xdai/address/0xb97FCb6adf4c4aF9981932a004e6CC47173d0Bfc/contracts) |
+| CPKFactory | [0xfC7577774887aAE7bAcdf0Fc8ce041DA0b3200f7](https://blockscout.com/poa/xdai/address/0xfC7577774887aAE7bAcdf0Fc8ce041DA0b3200f7/contracts) |
+| Uniswap V2 Router | [0x1C232F01118CB8B424793ae03F870aa7D0ac7f77](https://blockscout.com/poa/xdai/address/0x1C232F01118CB8B424793ae03F870aa7D0ac7f77/contracts) |
+| Uniswap V2 Factory | [0xA818b4F111Ccac7AA31D0BCc0806d64F2E0737D7](https://blockscout.com/poa/xdai/address/0xA818b4F111Ccac7AA31D0BCc0806d64F2E0737D7/contracts) |
+| Wrapped1155Factory | [0xDE6943f3717738038159a406FF157d4eb3238c1B](https://blockscout.com/poa/xdai/address/0xDE6943f3717738038159a406FF157d4eb3238c1B/contracts) |
+
+
+## Sokol Network
+
+| Contract | Address |
+| -------- | -------- |
+| [ConditionalTokens](https://github.com/gnosis/conditional-tokens-contracts) | [0x0Db8C35045a830DC7F2A4dd87ef90e7A9Cd0534f](https://etherscan.io/address/0xC59b0e4De5F1248C1140964E0fF287B192407E0C#code)     |
```

Comment:
Looks like a mainnet address slipped in here

Reply:
fixed